css/style.css

```diff
@@ -0,0 +1,122 @@
+body {
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 400;
+  text-align: center;
+  background:#fffef3;
+}
+
+a {
+  text-decoration: none;
+  font-weight: 600;
+  color: #988a81;
+}
+
+a:hover {
+  color: #bcb2a9;
+}
+
+nav {
+  font-size: 18px;
+  text-transform: uppercase;
+  margin-bottom: 25px;
+  display: inline-block;
+}
+
+.nav-container {
```

Comment:
Good use of classes to style.  I would argue that you could merge this class and the `nav` tag in both the HTML and CSS.